src/appmixer/salt/core/ListPostureGaps/ListPostureGaps.js

```diff
@@ -0,0 +1,43 @@
+const apiCall = (context, { offset, limit, severity = [], status = [] }) => {
+
+    const severityParam = severity.length > 0 ? `&severity=${severity.join(',')}` : '';
+    const statusParam = status.length > 0 ? `&status=${status.join(',')}` : '';
+
+    const baseUrl = context.config.baseUrl || 'https://api.secured-api.com';
+
+    return context.httpRequest({
+        method: 'GET',
+        url: `${baseUrl}/v1/apigovern/posture/gaps?limit=${limit}&offset=${offset}&details=true${severityParam}${statusParam}`,
+        headers: {
+            'Authorization': `Bearer ${context.auth.apiKey}`
+        }
+    });
+};
+
+module.exports = {
+    async receive(context) {
+        const { limit: limitTotal, severity, status } = context.messages.in.content;
```

Comment:
Add a newline after function declaration.
```suggestion
    async receive(context) {

        const { limit: limitTotal, severity, status } = context.messages.in.content;
```